_automating-configurations/api/create-workflow.md

```diff
@@ -72,12 +72,26 @@ PUT /_plugins/_flow_framework/workflow/<workflow_id>?update_fields=true
 You cannot specify both the `provision` and `update_fields` parameters at the same time.
 {: .note}
 
+If a workflow has been provisioned, you can update and reprovision the full template:
+
+```json
+PUT /_plugins/_flow_framework/workflow/<workflow_id>?reprovision=true
+{
+  <updated complete template>
+}
+```
+{% include copy-curl.html %}
```

Comment:
can be removed I assume

Reply:
I'll leave that to doc reviewers.  I see the `<placeholder>` stuff used in other copy/paste.

Reply:
Yeah I see that now in other locations

Reply:
We typically want to spell out everything in the documentation in order to avoid confusion. This request clearly shows the `reprovision` query parameter and specifies that it needs to be a _complete_ template, so we should keep this here. The template is very big so it's fine to use a descriptive placeholder.